### DISCUSSION TITLE
any plans for any documentation?

### DISCUSSION THREAD

**Discussion by o-az** [open-circle/valibot#322] [Q&A]
Trying to select between a few tools and found very little documentation for valibot. Curious when do you think most APIs will be documented?

**Comment by fabian-hiller** [comment]
We started to extend the API reference. Have a look at #287 for the current status.

If you are an experienced developer, our source code is well documented. So feel free to have a look at it for any API that is not yet documented on our website.

Also, feel free to create issues or discussions for any problems or questions you have. I usually answer in less than 24 hours.
